src/index.html

```diff
@@ -6,9 +6,30 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a class ="logo" href ="logo"><img src ="/src/images/logo.png" alt = "Logo"></a>
```

Comment:
1. no needs to add extra spaces when you add a tag attribute
2. check code styling recommendations and apply all over the page
https://github.com/myemets/layout_moyo-header/blob/master/checklist.md
```suggestion
      <a class="logo" href="logo">
        <img src="/src/images/logo.png" alt="Logo">
      </a>
```